src/components/molecules/notes-visualizer/NotesVisualizer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import "./NotesVisualizer.scss"
+
+
+type noteProps = {
+    editMode: boolean
+    currentNote: string
```

Comment:
A best practice for presentational components is to have a unique prop `data` that contains all the data that will be visualized. 
```typescript
type noteProps = {
    editMode: boolean
    data: {
       currentNote: string
     }
}
```